dbms/src/Common/TiFlashMetrics.h

```diff
@@ -110,8 +108,6 @@ namespace DB
         F(type_passthrough_none_compression_remote, {"type", "passthrough_none_compression_remote"}),                                               \
         F(type_passthrough_lz4_compression, {"type", "passthrough_lz4_compression"}),                                                               \
         F(type_passthrough_zstd_compression, {"type", "passthrough_zstd_compression"}))                                                             \
-    M(tiflash_schema_version, "Current version of tiflash cached schema", Gauge)                                                                    \
-    M(tiflash_sync_schema_applying, "Whether the schema is applying or not (holding lock)", Gauge)                                                  \
```

Comment:
Keeping this metrics maybe helpful for us locating bugs that the threads is stuck by schema sync

Reply:
Fixed